### PR TITLE
Update barrier_ctrl

### PR DESCRIPTION
**Pull request by BudzikM**
stopped_at was not used by anyone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1026,32 +1026,32 @@ The service represent devices like garage door openers, barriers, window protect
...
+in   | cmd.tstate.set           | string     |           | This command sets target barrier's state using value from `sup_tstates`. Event evt.state.report is sent as a confirmation.
 
 #### Interface props
 
-Name        | Value example | Description
-------------|---------------|-------------
-`stopped_at`| 30            | Stopped at exact position (percentage value).
+Name           | Value example | Description
+---------------|---------------|-------------
+`cur_pos`      | 30            | Current barrier position in percentage.
```
Comment (BudzikM) [inline README.md:1046]: Mobile team didnt have a chance to use it yet. @GenixPL

File: README.md [hunk 2]
```diff
@@ -1026,32 +1026,32 @@ The service represent devices like garage door openers, barriers, window protect
...
-Name             | Value example                  | Description
------------------|--------------------------------|-------------
-`sup_notiftypes` | audio, visual                  | supported notification-types, like siren, flashlight
-`sup_states`     | open, closed, closing, opening | supported states
-`sup_tstates`    | open, closed                    | supported target states
+Name             | Value example                 | Description
+-----------------|-------------------------------|-------------
+`sup_notiftypes` | audio, visual                 | Supported notification-types like siren, flashlight.
+`sup_states`     | closing, opening, stopped     | Supported transition states reflecting momentary barrier's state.
+`sup_tstates`    | open, closed, close, n%       | Supported target states where 'n' is an interger value withing range <0,100>.
```
Comment (BudzikM) [inline README.md:1054]: closed seems to be a type, but because of BC we cannot change it probably.

File: README.md (hunk 1 above)
Comment (GenixPL) [reply README.md:1046]: yup

File: README.md (hunk 2 above)
Comment (boguslaw-wojcik) [reply README.md:1054]: open - a noun, a state of being open
closed - a noun, a state of being closed
open - a verb, a command to open something
close - a verb, a command to close something

I think target state "close" doesn't make any sense.

**Comment by boguslaw-wojcik** [comment]
This PR is already outdated.

**Pull request closed without merging**